src/data/faq.json

```diff
@@ -1219,14 +1219,16 @@
                                 ]
                             },
                             {
-                                "title": "In-App statistics starting with release 1.11",
+                                "title": "In-App statistics",
                                 "anchor": "further_details",
                                 "active": false,
                                 "textblock": [
-                                    "Due to the high dynamics of the pandemic, the Corona-Warn-App displays important statistical key figures on current infections starting with the 1.11 version of the app. The figures displayed guide users to better assess the current situation and to adjust their own behavior.",
+                                    "Since version 1.11 the Corona-Warn-App displays important statistical key figures on current infections. The figures displayed guide users to better assess the current situation and to adjust their own behavior.",

```

Comment:
In the German version it says "`<b>Aktualisiert am 19.10.2022</b>`". The corresponding text is missing in the English version.

Reply:
Good catch, this must have gotten lost somehow. I was sure that was there. :(
Fixed. Thanks!